.github/workflows/ci.yml

```diff
@@ -35,6 +35,8 @@ jobs:
       - name: Get rust version
         id: rust-version
         run: echo "::set-output name=version::$(rustc --version)"
+      - name: Install Ninja
+        uses: seanmiddleditch/gha-setup-ninja@v4
```

Comment:
Will this pass our security check on GH? AFAIR, we enabled the white list enforcement for all repos last week.

Reply:
It'll need to be added, but it works in PR form just fine, it seems. I'll open a PR once this lands, as CI passing at that point isn't a blocker.